src/main/java/com/synerzip/ExpenseCalculation/User.java

```diff
@@ -0,0 +1,146 @@
+package com.synerzip.ExpenseCalculation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.NamedQuery;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.persistence.TableGenerator;
+
+import org.hibernate.annotations.NamedQueries;
+
+@Entity
+@NamedQuery(name = "findByEmail_id", query = "SELECT u FROM User u WHERE u.email_id= :email_id")
+// tableGenerator is used to start the id field with 1.
+@TableGenerator(name = "tab", initialValue = 0, allocationSize = 50)
+@Table(name = "user")
+public class User {
+
+	public User() {
+	}
```

Comment:
follow proper order of declaration. #codeconvention [see](http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141855.html#1852)

this comment for all